src/index.js

```diff
@@ -97,13 +100,17 @@ FlowWebpackPlugin.prototype.apply = function(compiler) {
         compilation.errors.push('Flow validation' + details)
     })
 
+    if(plugin.options.bundle) {
+        compiler.plugin('done', runCallback(false))
+    }else{
     /*
      * callbacks chosen because it is required
      * * to be done before or at time of 'compilation callback' - to avoid expensive compilation when type error present
      * * hook needs to be asynchronous
```

Comment:
Hi @daneshka, sorry for late response. Yes, I believe that `after-emit` phase is best for both running the flow checker and reporting the result (I've added support for user callbacks which required a bit of refactoring). I'd call the option `afterEmit`.

The patch is appreciated, but if you want just checking in your project, you can use this very simplistic checker (`webpack.config.js`):

```js
const flowPath = require.main.require('flow-bin')
const cp = require('child_process')

const flowPlugin = {
    apply: function(compiler) {
        compiler.plugin('after-emit', () => {
            cp.spawnSync(flowPath , [], { stdio: 'inherit' })
        })
    }
}

module.exports = {
    // ...
    plugins: [
        flowPlugin
    ],
}
```